public/index.php

```diff
@@ -23,7 +23,7 @@
 
 require_once dirname(__DIR__) . '/preload.php';
 
-if ($_ENV['YII_ENV'] === 'test') {
+if ($_ENV['YII_ENV'] === 'test' || getenv('YII_ENV') === 'test') {
```

Comment:
Should be no need for it since it's taken care of in https://github.com/yiisoft/demo/blob/master/preload.php#L12. If dotenv wasn't able to load it, that's quite bad but it should be fixed in preload anyway.

Reply:
We should use `getenv('YII_DEV')`. See https://mattallan.me/posts/how-php-environment-variables-actually-work/

Reply:
```suggestion
if (getenv('YII_ENV') === 'test') {
```